README.rst

```diff
@@ -17,4 +17,25 @@ Available states
 ------------
 
 Installs pydio package and adds pydio's official repositories.
+creates a mysql database with password from pillar
 
+``pydio.apache``
+----------------
+
+Installs apache configuration suitable for pydio
+
+``pydio.repo``
+--------------
+
+Installs the debian pydio repo
+included in pydio.sls
```

Comment:
You don't need to document states that aren't optional. In fact it could be confusing if someone thinks the repo functionality is optional and it uses it anyway.